File: 240307/학점 계산기/credit-calculator.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin>>n;
    double arr[n];

    double sum=0;
    double cnt=0;

    for(int i=0;i<n;i++){
        cin>>arr[i];
        sum+=arr[i];
        cnt++;
    }
    cout.precision(1);
    cout.fixed;
    cout<<sum/cnt<<"\n";
    if(sum/cnt>=4.0) cout<< "Perfect";
    else if (sum/cnt>=3.0) cout<<"Good";
    else cout<<"Poor";
return 0;
}

File: 240527/학점 계산기/credit-calculator.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);

        int n=sc.nextInt();
        int sum=0,cnt=0;
        double avg;
        double [] arr=new double[n];

        for(int i=0;i<n;i++){
            arr[i]=sc.nextDouble();
            sum+=arr[i];
            cnt++;
        }
        avg=(double) sum/cnt;

        if(avg>=4.0)
        System.out.print("Perfect");
        else if(avg>=3.0)
        System.out.print("Good");
        else
        System.out.print("Poor");
        // 여기에 코드를 작성해주세요.
    }
}